modules/homepage/src/main/frontend/src/themePage/Navbars/AdminNavbarLinks.jsx

```diff
@@ -189,6 +189,8 @@ function HeaderLinks (props) {
             className: classes.successSnackbar,
           },
         }}
+        sx={{ ml: "130px" }}
+        anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
```

Comment:
Ideally we would fix the z-index of the snack bar message instead, but with all the custom styling of the dashboard this may be less pleasant.

Instead, I like the idea of changing the position, but I would suggest `horizontal: 'right'` instead of `center` for 2 reasons:
* to avoid the need for sidebar accomodations
* to display it where the origin button is (top right).

```suggestion
        anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
```